src/resource-archive/index.ts

```diff
@@ -126,6 +126,13 @@ class Watcher {
       requestUrl.host === this.firstUrl.host &&
       requestUrl.port === this.firstUrl.port;
 
+    // We only need to capture assets that will render when the DOM snapshot is rendered,
+    // so we only need to handle GET requests.
+    if (!request.method.match(/get/i)) {
+      await this.clientSend(request, 'Fetch.continueRequest', { requestId });
+      return;
+    }
```

Comment:
Seems we could do this very-first in the function, but I don't feel strongly about that.

Reply:
Yeah I was kind of unsure where to put this in the function, tbh! I think I put it there because there was similar code to bail nearby, but I'll give it another look.

Reply:
@skitterm I did move it to the top of the function. Thanks!